notes-webapp/src/main/resources/locale/portlet/notes/notesPortlet_en.properties

```diff
@@ -55,7 +55,7 @@ popup.confirmation.delete.draft=Delete draft
 popup.msg.confirmation.DeleteInfo1=Are you sure you want to send this note to the trash? {0}
 popup.msg.confirmation.DeleteInfo2=Please note:
 popup.msg.confirmation.DeleteInfo4=The links to this page will be broken when this page is removed.
-popup.msg.confirmation.DeleteInfo5=This note is child of the note: {1}
+popup.msg.confirmation.DeleteInfo5=This note is child of the note: {0}
```

Comment:
This fix should be made on Crowdin, pleasee see https://developer.exoplatform.org/#id-how-to-update-an-existing-translation-string

Reply:
@hghorbel translation added in crowdin to fix it : https://crowdin.com/translate/exo-platform/37547/en-en#7013739